blog/2023-03-16-financial-revolution-powered-by-the-blockchain/financial-revolution-powered-by-the-blockchain.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Financial Revolution powered by blockchain Technology"
+description: In this article,we will look at how blockchain technology can be leveraged to reimagine money to create the condition for prosperity for everyone .
+authors:
+  - name: Ronald Nzioki
+    title:Technical Writer
+    url: https://github.com/rony23-byte
+    image_url: 
+tags: [defi,blockchain,revolution]
+image: https://dl.airtable.com/.attachmentThumbnails/a7e530eb72ac8f30f37c0a3447ef0e7d/72e944da
+hide_table_of_contents: true
+
+slug: /tutorial/financial revolution powered by blockchain tech
+
+
+---
+`
+
+**Introduction​**
+
+In this article,we will look at how blockchain technology can be leveraged to reimagine money to create the condition for prosperity for everyone.Lets dive in
+
+The genie is out of the bottle, ready to serve us in transforming the economic power grid.
+
+What would it take for blockchain technology to create a utopian future that would truly democratize finance?
+
+The second iteration of the internet brought the world closer together, making it easier for people to trade and connect. Again, however, the need for trust brought about the rise of various intermediaries.
+
+These intermediaries were doing a relatively okay job. However, cracks started showing up when too-big-to-fail financial institutions increased their appetite for risk by creating and investing in Mortgage-backed securities; thereby causing financial distress to average Joe's who had no clue of the undertakings of these banks; centralized institutions could, therefore, not be trusted. As if not enough, there was a lot of friction when sending money; this excluded so many people from financial services, locking a large population from accessing financial services.
+
+What if we did not have to subscribe to an Orwellian future?
+
+
+**Prerequisites**
```

Comment:
Use subheadings (H2: ##) to break down your explanations within the body of the tutorial.